README.md

```diff
@@ -127,6 +127,29 @@ circumstances it may be beneficial to immediately render the application and han
 component-by-component basis. The React Native Client SDK's asynchronous mode allows this by passing the
 optional `async` prop when connecting with the `FFContextProvider`.
 
+## Streaming and polling
```

Comment:
Is it worth adding an explicit section on "no polling or streaming" ? Something like https://github.com/harness/ff-javascript-client-sdk#no-streaming-or-polling

Reply:
Yeah, I think we can have something like that here as well. I'll get that in tomorrow (I need to open a PR to update the React SDK to the latest).